<ui version="4.0" >
 <class>prefs_projectmain</class>
 <widget class="QWidget" name="prefs_projectmain" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>375</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>11</number>
   </property>
   <property name="topMargin" >
    <number>11</number>
   </property>
   <property name="rightMargin" >
    <number>11</number>
   </property>
   <property name="bottomMargin" >
    <number>11</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <widget class="QLabel" name="textLabel2" >
     <property name="text" >
      <string>Template diectory:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="textLabel2_2" >
     <property name="text" >
      <string>PO directory:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="kcfg_PoBaseDir" />
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="kcfg_PotBaseDir" />
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="kcfg_GlossaryTbx" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Glossary:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Language:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="LangCode" />
   </item>
   <item row="4" column="1" >
    <widget class="QLineEdit" name="kcfg_LangCode" />
     <property name="visible" >
      <bool>false</bool>
     </property>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>LangCode</sender>
   <signal>activated(QString)</signal>
   <receiver>kcfg_LangCode</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>359</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel" >
     <x>359</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
